IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ElectricFlowStorageDeviceTypeInductor/DocPropertySet.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_ElectricFlowStorageDeviceTypeInductor" UniqueId="7a6ea86a-c228-d78b-56d8-38a6ae61f7b1" ApplicableType="IfcElectricFlowStorageDevice/INDUCTOR" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="Inductance_26NOHc5AHAsRdIVvvqQ_g" />
+		<DocProperty xsi:nil="true" href="NumberOfPhases_0ZOkcOH14AAoaednCd9ms" />
+		<DocProperty xsi:nil="true" href="InsulationStandardClass_1TFB_0qTyHuO00025QrEV" />
```

Comment:
all  Electrical Device properties should be used in conjunction with PSet_ElectricalDeviceCommon, for initial base properties.
therefore this is already on that PSet and should be used there. along with Rated Current and voltage values

Reply:
Agree with you! Has been erased. 840d7b293d3051eca8050639884528e00dfc5d4e